source/frontend/src/features/properties/map/agreement/detail/AgreementView.tsx

```diff
@@ -33,7 +33,12 @@ export const AgreementView: React.FunctionComponent<IAgreementViewProps> = ({
           <EditButton title="Edit agreements file" onClick={onEdit} />
         ) : null}
       </StyledEditWrapper>
-      {agreements.length === 0 && <StyledNoData>No agreements on record</StyledNoData>}
```

Comment:
didn't I call this out on the original PR? eh.

Reply:
You did indeed. I figured the AC didnt mention it, so it was more of a UI suggestion than a requirement :)